pinboard_config.example.rb

```diff
@@ -0,0 +1,63 @@
+{
+  secret_base_url: 'secret:abc123/',
+  rss_base: "https://feeds.pinboard.in/rss/",
+  user: "u:whoami/",
+  public_tags: [
+    '','t:via:popular/','t:Programming/','t:ifttt','t:ruby'
+  ],
+  private_tags: [
+    't:Bookmarks_Bar','from:instapaper','from:twitter','from:twitter_favs','starred','untagged','private','toread'
```

Comment:
Line is too long. [114/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.